src/pages/createlist/CreateListPage.tsx

```diff
@@ -29,6 +29,13 @@ export const CreateListPage:FC = () => {
   const { showSuccessMessage, showErrorMessage } = useMessages();
   const { showCommandError } = useKeyCommandsMessages();
 
+  const {
+    showConfirmCancelEditModal,
+    continueNavigation,
+    keepEditHandler,
+    setShowConfirmCancelEditModal
```

Comment:
`showConfirmCancelEditModal` and `setShowConfirmCancelEditModal` are poorly named. You're not setting the Modal, you're setting the Modal's visibility. Say that with their names: `isModalVisible`, `setIsModalVisible`.